package.json

```diff
@@ -125,6 +124,7 @@
     "vega": "^5.13.0",
     "vega-lite": "^4.13.0",
     "vitessce": "^0.2.1",
-    "webpack": "^4.43.0"
+    "webpack": "^4.43.0",
+    "react-dropzone": "^11.3.1"
```

Comment:
was this an intended move?I am just asking because of the recent socket io issues we had in master

Reply:
It was, things should be good now.